google-http-client/src/main/java/com/google/api/client/http/Scope.java

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright 2019 Google LLC
+ *
+ * <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
```

Comment:
You don't need `<p>` tags in the license

Reply:
Whoops, it appears the formatter got overzealous. I will fill all of these.